### ISSUE TITLE
Debug disable not work EntityDamageByEntityEvent

### ISSUE THREAD

**Issue by andris155** [SirBlobman/CombatLogX#143]
**Describe the bug**
Debug disable not work

**To Reproduce**
How can I see the issue for myself? Please explain how it happened.
Set debug: false and cause EntityDamageByEntityEvent.

**Expected behavior**
Disable this Debug.

**Screenshots**
If applicable, add screenshots or videos to help explain your problem.

**Minecraft Version**
 - 1.16.1

**CombatLogX Version:**
 - v10.0.0.0.Beta-256

**Additional context**
```
[18:41:41 INFO]: [CombatLogX] [Debug] Detected EntityDamageByEntityEvent...
[18:41:41 INFO]: [CombatLogX] [Debug] Damaged entity is not a player.
[18:41:41 INFO]: [CombatLogX] [Debug] Detected EntityDamageByEntityEvent...
[18:41:41 INFO]: [CombatLogX] [Debug] Damaged entity is not a player.
[18:41:41 INFO]: [CombatLogX] [Debug] Detected EntityDamageByEntityEvent...
[18:41:41 INFO]: [CombatLogX] [Debug] Damaged entity is not a player.
[18:41:41 INFO]: [CombatLogX] [Debug] Detected EntityDamageByEntityEvent...
[18:41:41 INFO]: [CombatLogX] [Debug] Damaged entity is not a player.
[18:41:41 INFO]: [CombatLogX] [Debug] Detected EntityDamageByEntityEvent...
[18:41:41 INFO]: [CombatLogX] [Debug] Damaged entity is not a player.
```

**Comment by andris155** [comment]
Thanks. But this build failed on the Jenkins server. :(

**Comment by SirBlobman** [comment]
> Thanks. But this build failed on the Jenkins server. :(

This should be fixed in commit [d6d6b55ddabbf90ae8fc4665322ae216c9a6f007](https://github.com/SirBlobman/CombatLogX/commit/d6d6b55ddabbf90ae8fc4665322ae216c9a6f007)

**Comment by andris155** [comment]
Thanks!
